tests/cases/conformance/classes/constructorDeclarations/superCalls/derivedClassParameterProperties.ts

```diff
@@ -56,7 +56,7 @@ class Derived7 extends Base {
     constructor(y: string) {
         this.a = 3;
         this.b = 3;
-        super(); // error
+        super();

```

Comment:
Still an error (because of the `this` reference on the line above)

Reply:
I had removed a bunch of comments like this one since they were falling out of sync with the `.errors.txt` baselines. I'll go ahead and finish removing the rest of them from this file -- mostly `// ok`s.

Reply:
I had removed a bunch of comments like this one since they were falling out of sync with the `.errors.txt` baselines. I'll go ahead and finish removing the rest of them from this file -- mostly `// ok`s.